1906-10-08.xml

```diff
@@ -58,7 +59,7 @@
                         <p>Mable, Todd &amp; Bard, High Holborn, London.</p>
                     </div>
                 </div>
-                <div type="section" scope="advertisement">
+                <div type="section" scope="advertisement"></div>
```

Comment:
move this `</div>` tag, which closes the section, all the way to the end of the ads on this page, i.e. just before the `</div>` tag that closes the page.